core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java

```diff
@@ -0,0 +1,29 @@
+package com.arcrobotics.ftclib.geometry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class Transform2dTest {
```

Comment:
Only one test?

Reply:
That's the only thing present in wpilib testing, wasn't sure if we needed more testing than they do.

Reply:
Just go ahead and add some more tests for the other methods. Quick ones will work fine.

Reply:
Will do.